artemis-commons/src/main/java/org/apache/activemq/artemis/utils/Base64.java

```diff
@@ -1,1619 +1,61 @@
-/**
- * This file is released into the public domain.
```

Comment:
The source and binary assembly LICENCE files will both be needing an update due to the removal.

Reply:
See #5240.